code/src/pages/products/ProductsPage.test.tsx

```diff
@@ -5,6 +5,15 @@ import { useGetProductsQuery } from "@/store/api/productsApi";
 import { PaginationParams, RTKQueryReturnState } from "@/types/common";
 import renderWithProviders from "@/utils/render-with-providers/renderWithProviders";
 
+jest.mock("@/store/api/cartApi", () => ({
+  useAddToCartMutation: jest.fn(() => [jest.fn(), {}])
+}));
+
+jest.mock("@/hooks/use-snackbar/useSnackbar", () => ({
+  __esModule: true,
+  default: jest.fn(() => ({ openSnackbar: () => {} }))
+}));
```

Comment:
Same

Reply:
Test falls without it, already explained in other comment